cmd/storage/main.go

```diff
@@ -81,20 +83,25 @@ func init() { //nolint:gochecknoinits
 	flag.IntVar(&cfg.extapi.Port, "extapi.port", def.ExtAPIPort, "serve external API on `port` (>0)")
 	flag.StringVar(&cfg.extapi.BasePath, "extapi.basepath", def.ExtAPIBasePath, "serve external API on `path`")
 	flag.StringVar(&cfg.kasse.Endpoint, "kasse.endpoint", def.KasseEndpoint, "endpoint online kasse")
+	flag.BoolVar(&dbChange, "d", false, "change database from postgres to memdb")
+	flag.BoolVar(&timeCheck, "t", false, "check time")
 
 	log.SetDefaultKeyvals(
 		structlog.KeyUnit, "main",
 	)
 }
 
 func main() { //nolint:gocyclo
+
 	flag.Usage = func() {
 		fmt.Printf("Usage of %s:\n", cmd)
 		flag.PrintDefaults()
 		fmt.Print(goose.Usage)
 	}
 	flag.Parse()
 
+	checkTime(timeCheck)
```

Comment:
Use `timeCheck` around  `checkTime()`:
```
if timeCheck {
    checkTime()
}
```
This function should check system time only. `main` checks flags and calls functions accordingly.